ruby/deepgram-language-detection/index.rb

```diff
@@ -0,0 +1,39 @@
+require "uri"
+require "net/http"
+require "openssl"
+require "json"
+
+
+def main(req, res)
+  fileUrl = nil
+  begin
+    payload = JSON.parse(req.payload)
+    fileUrl = payload["fileUrl"]
+  rescue Exception => err
+      puts err
+      raise "Payload is invalid."
```

Comment:
instead of raising an error, please make sure to call `res.json()` and return an informative error response as defined in the issue.